interface_specifications/pe_openapi/apis/v1/d4_pe_presentations_get.md

```diff
@@ -0,0 +1,59 @@
+### Presentations GET
+
+`GET /pe/v1/presentations?offset=0&limit=20`
```

Comment:
How do we know which ones to retrieve now? In this call we do not have the presentation_id(s) and we don't have the thread id as input. We either need the thread_id, or we need to have some other session/persistence that allows us to bind the caller to this information

Reply:
fixed, deleted